features/config.feature

```diff
@@ -50,3 +50,39 @@ Feature: pro config sub-command
             | xenial  | lxd-container |
             | jammy   | lxd-container |
             | mantic  | lxd-container |
+
+    Scenario Outline: Proxy information is redacted when using pro config show
```

Comment:
@dheyay I think we can already reuse one of the existing proxy tests for this check, just to avoid creating another instance just for this feature

Reply:
Sure, will find a better place for this test